apps/etterlatte-gyldig-soeknad/src/main/kotlin/no/nav/etterlatte/gyldigsoeknad/NySoeknadRiver.kt

```diff
@@ -0,0 +1,113 @@
+package no.nav.etterlatte.gyldigsoeknad
+
+import com.fasterxml.jackson.databind.JsonMappingException
+import io.ktor.client.plugins.ResponseException
+import kotlinx.coroutines.runBlocking
+import no.nav.etterlatte.gyldigsoeknad.client.BehandlingClient
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.event.FordelerFordelt
+import no.nav.etterlatte.libs.common.event.GyldigSoeknadVurdert
+import no.nav.etterlatte.libs.common.event.SoeknadInnsendt
+import no.nav.etterlatte.libs.common.event.SoeknadInnsendtHendelseType
+import no.nav.etterlatte.libs.common.innsendtsoeknad.common.SoeknadType
+import no.nav.etterlatte.libs.common.logging.getCorrelationId
+import no.nav.etterlatte.libs.common.rapidsandrivers.correlationId
+import no.nav.etterlatte.rapidsandrivers.ListenerMedLogging
+import no.nav.etterlatte.sikkerLogg
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import org.slf4j.LoggerFactory
+
+internal class NySoeknadRiver(
```

Comment:
Dette er tidligere `FordelerRiver` fra `etterlatte-fordeler`. Forenklet kraftig og flyttet hit for å bevare dagens flyt (enn så lenge)